### DISCUSSION TITLE
release faild on android

### DISCUSSION THREAD

**Discussion by wkmhv** [unoplatform/uno#18791] [General]
no problem in debug.
Fails on release

![grafik](https://github.com/user-attachments/assets/4bd9ac21-27d4-4446-9338-d37fbf7324a2)
![grafik](https://github.com/user-attachments/assets/d1af597f-07a1-4f80-b351-f490f40b3457)

**Comment by jeromelaban** [comment]
Thanks for the report. Could you [provide a binlog of your build](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?

**Reply by wkmhv** [reply]
[binlog.zip](https://github.com/user-attachments/files/17784214/binlog.zip)

**Reply by jeromelaban** [reply]
Thanks for the report. This is an issue of the .NET SDK, where the tooling does not handle spaces and/or accentuated characters in the path.

```
[CommunityToolkit.Mvvm.dll] Running: C:\Program Files\dotnet\packs\Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm\8.0.11\Sdk\..\tools\mono-aot-cross.exe --response="C:\Users\Werner König\AppData\Local\Temp\tmp4AAD.tmp"
```

I've opened this issue here: https://github.com/dotnet/android/issues/9523. The only known way around this is to have a user folder that does not have spaces and accentuated characters, but it requires a lot of changes in your environment. Make sure to comment on the issue I referenced.
